hbase-server/src/main/java/org/apache/hadoop/hbase/monitoring/TaskMonitor.java

```diff
@@ -85,14 +85,14 @@ public static synchronized TaskMonitor get() {
   }
 
   public MonitoredTask createStatus(String description) {
-    return createStatus(description, false);
+    return createStatus(description, true);
   }
 
-  public MonitoredTask createStatus(String description, boolean ignore) {
-    return createStatus(description, ignore, false);
+  public MonitoredTask createStatus(String description, boolean show) {
```

Comment:
Nitpick: you call this `show` here and `track` below. Can you use one name? (Track maybe better)

While you are editing these methods could to also add small javadoc to explain what the param does (like you added elsewhere in TaskGroup)